app/representation.py

```diff
@@ -1,43 +1,132 @@
+import random
+
 def venueRecord(biz, **details):
     # biz is the response object from the Yelp Search API
+
+    from collections import OrderedDict
+    # h is derived from the providers, but for the main body of the record.
+    # h is for header.
+    h = {
+      "url"        : "https://mozilla.org",
+      "description": {},
+      "categories" : OrderedDict(),
+      "images"     : [],
+      "hours"      : [],
+    }
     providers = {}
-    images = []
-    hours = []
+    
+    # Yelp.
     if "yelp" in details:
       info = details["yelp"]
       providers["yelp"] = {
-        "rating": biz.rating,
-        "reviewCount": biz.review_count,
-        "ratingMax": 5,
-        "description": biz.snippet_text,
-        "url": biz.url
+        "rating"          : biz.rating,
+        "totalReviewCount": biz.review_count,
+        "ratingMax"       : 5,
```

Comment:
The schema doesn't have a ratingMax, and instead has rating as a "normalized 0-5 w/ half values"